pkg/term/term_linux_test.go

```diff
@@ -0,0 +1,120 @@
+//+build linux
+
+package term
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
I suppose that would work, but I think I prefer the flag, because it means the set of tests that runs won't be sensitive to the environment. It's nice to avoid scenarios where the tests pass for one person but fail for another person running the same comment.

containerd uses this flag to enable/disable tests that require root privileges.